Tests/MessageTests.swift

```diff
@@ -11,4 +12,34 @@ class MessageTests: XCTestCase {
         XCTAssertEqual(Message.Kind.shorterTomorrowMoreThanAMinute, Message.Kind(sunPhase: .night, yesterdayDaylightLength: 0, todayDaylightLength: 61, tomorrowDaylightLength: 0))
         XCTAssertEqual(Message.Kind.shorterTomorrowLessThanAMinute, Message.Kind(sunPhase: .night, yesterdayDaylightLength: 0, todayDaylightLength: 60, tomorrowDaylightLength: 0))
     }
+
+    func testNoAttributesColoredMessage() {
+        let content = "Hello mom."
+        let coloredPart = "mom"
+
+        let message = Message(format: "Hello **mom**.")
+        XCTAssertEqual(message.content, content)
+        XCTAssertEqual(message.coloredPart, coloredPart)
+
+        let range = (content as NSString).range(of: coloredPart)
+        let attributedString = NSMutableAttributedString(string: content)
+        attributedString.addAttribute(NSForegroundColorAttributeName, value: UIColor.red, range: range)
+
+        XCTAssertEqual(message.attributedString(withTextColor: UIColor.red), attributedString)
```

Comment:
Aha I see, it's one of our own methods ✌️ Just wondering then if we should test this method in a separate test. One test for the .colouredPart to be recognized correclty, and one to apply the right attribute extravaganza.